tests/System.Numerics.Tensors.Tests/System.Numerics.Tensors.Tests.csproj

```diff
@@ -17,10 +18,7 @@
     <ProjectReference Include="..\..\src\System.Numerics.Tensors\System.Numerics.Tensors.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemCompilerServicesUnsafeVersion)" />
-  </ItemGroup>
-  <!-- register for test discovery in Visual Studio -->
-  <ItemGroup>
+    <!-- register for test discovery in Visual Studio -->
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
We may be able to remove this now that https://github.com/Microsoft/vstest/issues/472 is fixed.

Reply:
Removed.